packages/eslint-plugin/src/rules/ban-tslint-comment.ts

```diff
@@ -36,6 +36,11 @@ export default util.createRule({
       Program(): void {
         const comments = sourceCode.getAllComments();
         comments.forEach(c => {
+          // edge case to support
+          if (c.type === AST_TOKEN_TYPES.Shebang) {
+            return;
+          }
```

Comment:
Could you add a unit test for this and any other rule that calls `sourceCode.getAllComments` please? Now that we support shebangs, any rule that interacts with all comments should be tested to make sure they don't cause it to crash, report erroneously, etc.

Reply:
Yeah, I can do that 👍